src/config/external-applications.md

```diff
@@ -82,3 +82,26 @@ example, to build the control package, you must install the `openblas-devel`,
 
 To use MATLAB's help browser, live scripts, add-on installer, and simulink,
 install the `libselinux` package.
+
+## Steam
+
+Steam can be installed either via a native package, which requires [enabling the
+"nonfree" repository](../xbps/repositories/index.md###nonfree), or via
+[Flatpak](##flatpak). Dependency listings for different platforms and
+troubleshooting information for the native package can be found in its
+[Void-specific documentation file](./package-documentation/index.html), so the
+remainder of this section will deal with potential issues faced by Flatpak
+users.
+
+If you are using a different drive to store you game library run the following
+commmand, where `<storage>` is the mount point of your drive:
+
+`$ flatpak override com.valvesoftware.Steam --filesystem=<storage>`
+
+If sound is not working when Steam is installed in this manner, creating and
+running the following script to start Steam may fix the issue:
+
+```
+#!/bin/sh
+pulseaudio --start flatpak run com.valvesoftware.Steam
+```
```

Comment:
```suggestion
If sound is not working when Steam is installed with Flatpak, [PulseAudio](link to pulse) auto-activation might not be working correctly. Make sure PulseAudio is running before launching Steam.
```

Reply:
Do you think this advice is complete enough to get rid of the script that was included in the wiki? I'm not super familiar with pulse auto-activation, but my initial assumption is that it would be useful to keep the script (or maybe just the command) for users whose audio is not working even with pulse installed.

Reply:
I think if we recommend this at all, it should go in the PulseAudio section. But I think I'd rather see a recommendation for launching a terminal and running `pulseaudio --daemonize=no --exit-idle-time=-1` or something of the sort.

Reply:
I understand. I'll see about adding something like that to the PulseAudio page.

Reply:
Just opened a PR: #565.